manifest.yaml

```diff
@@ -1,65 +1,128 @@
-os-version-required: "^0.2.13"
-os-version-recommended: "^0.2.13"
-
-# Port mappings from tor ports to the container port
-ports:
-  - internal: 3000
-    tor: 80
-    # if lan is standard, it will proxy HTTP traffic from 443 to the internal port
-    lan: standard
-
-# Do not change this
-image:
-  type: tar
-
-# specifies where on the service's file system its persistence directory should be mounted prior to service startup
-mount: /root
-
-# Directory you would like to expose to dependent services, specified relative to the root of the persistence directory.
-# This will be a read-only filesystem for the dependent service
-public: public
-
-# Directory you would like to expose to dependent services, specified relative to the root of the persistence directory.
-# This will be a read-write filesystem for the dependent service
-shared: shared
-
-# Hidden Service version you would like to use for your services .onion url. Allowed options are "v2" and "v3"
-hidden-service-version: v3
-
-# This is a key value map specifying dependency services that you need to be able to function
-# The keys are the service id's on which you depend.
-# NOTE: if you are developing a standalone service, you may delete this entire stanza
+assets:
+  license: LICENSE
+  icon: icon.png
+  instructions: instructions.md
+  docker-images: image.tar
+main:
+  type: docker
+  image: main
+  entrypoint: docker_entrypoint.sh
+  args: []
+  mounts:
+    main: /root/
+    compat: /mnt/assets
+  io-format: yaml
+health-checks: {}
+config:
+  get:
+    type: docker
+    image: compat
+    system: true
+    entrypoint: compat
+    args: 
+      - config
+      - get
+      - /root/start9/config.yaml
+      - /mnt/assets/config_spec.yaml
+    mounts:
+      compat: /mnt/assets
+      main: /root
+    io-format: yaml
+  set:
+    type: docker
+    image: compat
+    system: true
+    entrypoint: compat
+    args:
+      - config
+      - set
+      - /root
+      - /mnt/assets/config_rules.yaml
+    mounts:
+      main: /root
+    io-format: yaml
+properties:
+  type: docker
+  image: compat
+  system: true
+  entrypoint: compat
+  args:
+    - properties
+    - /root
+  mounts:
+    main: /root
+  io-format: yaml
+volumes:
+  main:
+    type: data
+  compat:
+    type: assets
+interfaces:
+  main:
+    name: Browser Interface
+    description: The main Thunderhub browser interface
+    tor-config:
+      port-mapping:
+        3000: "80"
+    lan-config:
+      3000:
+        ssl: true
+        mapping: 443
+    ui: true
+    protocols:
+      - tcp
+      - http
 dependencies:
   lnd:
-    # The version range that is acceptable for this dependency
     version: ">=0.12.1 <0.14.0"
-    # Description of the dependency relationship
     decription: Thunderhub is an LND Manager
-    # If set to true it will mount the public directory of the dependency to $mount/start9/public/$dependencyId
     mount-public: true
```

Comment:
i should not have resolved this comment - this configuration is wrong for dependencies. mount public/shared no longer exists. there are many keys missing, including dependency check and autoconfigure.